src/mdbook.rs

```diff
@@ -3,13 +3,131 @@ pub fn handle_files(
     files: &[crate::FileMode],
 ) -> crate::Result<Vec<ft_api::bulk_update::Action>> {
     let mdbook_config = mdbook::config::Config::default();
+
     let book = mdbook::MDBook::init(config.root.as_str())
         .create_gitignore(true)
         .with_config(mdbook_config)
         .build()
         .expect("Book generation failed");
 
-    println!("{:#?}", book.book);
+    // TODO: Need to discuss it with amitu about appending "src"
+    let root_path = std::path::Path::new(config.root.as_str()).join("src");
+    let mut actions = vec![];
+    for file in files.iter() {
+        actions.append(&mut self::handle(
+            &file,
+            &root_path.to_string_lossy(),
+            config.collection.as_str(),
+        )?);
+    }
+
+    actions.push(self::index(&book.book, config)?);
```

Comment:
We can remove this altogether, as index can only change when `SUMMARY.md` or `title_page.md` has changed.